wicket-core/src/main/java/org/apache/wicket/DefaultExceptionMapper.java

```diff
@@ -56,8 +56,15 @@ public IRequestHandler map(Exception e)
 			Response response = RequestCycle.get().getResponse();
 			if (response instanceof WebResponse)
 			{
-				// we don't want to cache an exceptional reply in the browser
-				((WebResponse)response).disableCaching();
+				try
```

Comment:
@martin-g I'm not sure thsi is the best way to fix this. Other way would be have silent implementations in 

![image](https://github.com/apache/wicket/assets/462655/0f8b349a-741a-4719-9fba-f2503d934127)

But that  sound not correct.

Reply:
IMO a better way would be to check with [WebResponse.isHeaderSupported()](https://github.com/apache/wicket/blob/a15fbea9bea4e40684997f151e8f8957e0e33e3f/wicket-request/src/main/java/org/apache/wicket/request/http/WebResponse.java#L63) before calling `disableCaching()`.

Reply:
> IMO a better way would be to check with [WebResponse.isHeaderSupported()](https://github.com/apache/wicket/blob/a15fbea9bea4e40684997f151e8f8957e0e33e3f/wicket-request/src/main/java/org/apache/wicket/request/http/WebResponse.java#L63) before calling `disableCaching()`.

Oh! That is way better. I somehow missed this method!